static/specification.html

```diff
@@ -639,35 +671,55 @@ <h2>
         [SecureContext, Exposed=Window]
         interface MonetizationEvent : Event {
           constructor(DOMString type, MonetizationEventInit eventInitDict);
-          readonly attribute DOMString amount;
-          readonly attribute DOMString assetCode;
-          readonly attribute octet assetScale;
+          readonly attribute DOMString? amount;
+          readonly attribute DOMString? assetCode;
+          readonly attribute octet? assetScale;
           readonly attribute DOMString? receipt;
+          readonly attribute MonetizationCurrencyAmount amountSent;
           readonly attribute USVString paymentPointer;
           readonly attribute USVString? incomingPayment;
         };
 
         dictionary MonetizationEventInit : EventInit {
-          required DOMString amount;
-          required DOMString assetCode;
-          required octet assetScale;
+          required DOMString? amount;
+          required DOMString? assetCode;
+          required octet? assetScale;
           required DOMString? receipt;
+          required PaymentCurrencyAmount amountSent;
```

Comment:
`PaymentCurrencyAmount` or `MonetizationCurrencyAmount`?

Reply:
I believe this is still `PaymentCurrencyAmount` as it's a dictionary used to construct the event.

We may need someone with a deeper knowledge of browser internals like @sidvishnoi or @marcoscaceres to help here

Reply:
A dictionary can NOT be used as an attribute, so whether it's in namespace/scope to be used is a moot point.  
There is a lot of discussion around this in the issues, originally brought up by @sidvishnoi

Reply:
Oh, nevermind, in a different view I can see this is for MonetizationEventInit

Reply:
![image](https://user-images.githubusercontent.com/525211/204744512-59999474-085b-4a52-8dfb-dbb96459767e.png)

Respec compiles just fine